dotnet-authserver/src/TeacherIdentity.AuthServer/Models/User.cs

```diff
@@ -36,4 +36,6 @@ public class User
     public string? MobileNumber { get; set; }
     public MobileNumber? NormalizedMobileNumber { get; set; }
     public bool TrnLookupSupportTicketCreated { get; set; }
+
```

Comment:
Nit: don't need the newline here

Reply:
lint fails if the newline is removed. Running `dotnet format` adds the newline in